README.md

```diff
@@ -1,4 +1,77 @@
-# hubrise-webclient
-A pet JS/HTML project to see new HubRise orders in a web page
+<a href="https://www.hubrise.com/"><img src="./screenshots/hubrise-logo.png" align="left" height="48"></a>
+<br><br><br>
 
-[Demo](http://philatist-fish-74534.netlify.com)
+# HubRise {{app_name}} Template
+
+{{app_name}} is an app that prints new orders received by your HubRise location, with just a few lines of code.
+
+You can fork this repository and make changes to the code to create your own app. 
+
+
+## Try the app
+
+First, you need to [create a HubRise account](https://manager.hubrise.com/signup). It's free and it only takes a minute. 
+
+Then, simply [open the app](http://philatist-fish-74534.netlify.com) in your browser. 
+
+You will be asked to connect the app to your HubRise account. 
+Granting permissions is a necessary step in every OAuth2.0 authentication process, and requires the user to manually **Allow** the app access. 
+
+![OAuth page granting access to the app](./screenshots/oauth-page.png)
+
+You will then see a single-page app that displays the list of the latest orders received by your HubRise location. 
+Each line in the logs displays the time, id, status, and total price of the order.
+
+![Single-page app with the orders from HubRise](./screenshots/logs-page.png)
+
+Clicking on one of the logs reveals additional information about the order, such as the customer's details and the purchased products. 
+
+![Modal with additional information about the order](./screenshots/single-order-modal.png)
+
+
+## What this app is NOT
```

Comment:
@antmonnier In the issue, you mentioned that the user might want to add a DB. What would be the purpose of that in this case?

Reply:
Probably if the user was to extend this app, he would feel the need for a DB at some point. But maybe not even worth mentioning in this readme.

Reply:
OK fine!